clear.lua

```diff
@@ -9,10 +9,8 @@ function HandleClearCommand(Split, Player)
     if Player:HasPermission("core.admin.clear") then
         local InventoryCleared = false;
         local ClearInventory = function(OtherPlayer)
-            if (OtherPlayer:GetName() == Split[2]) then
-                OtherPlayer:GetInventory():Clear()
-                InventoryCleared = true
-            end
+            OtherPlayer:GetInventory():Clear()
+            InventoryCleared = true
```

Comment:
But the likelihood of that happening is very small, and the usability benefit of being able to use partial names IMO outweights the possibility of doing the command to the wrong person.